docs/application/native/guides/multimedia/image-classification.md

```diff
@@ -0,0 +1,211 @@
+# Image Classification
+
+Image classification is one of the main features of the Media Vision Inference API. You can [classify](#classify) an image, which belongs to the corresponding labels. For example, to classify an image consisting of a food item, you can apply a food classification model while doing inference of the decoded image data.
+
+## Prerequisites
+
+To enable your application to use the media vision inference functionality:
+
+1. To use the functions and data types of the Media Vision Inference API (in [mobile](../../api/mobile/latest/group__CAPI__MEDIA__VISION__INFERENCE__MODULE.html) and [wearable](../../api/wearable/latest/group__CAPI__MEDIA__VISION__INFERENCE__MODULE.html) applications), include the `<mv_inference.h>` header file in your application.
+
+   In addition, you must include the `<image_util.h>` header file to handle the image decoding tasks, or the `<camera.h>` header file to provide the preview images:
+
+   ``` c
```

Comment:
Global comment:
There should be no space between back-ticks and language identifier. Kindly remove the space between them.

Reply:
@rishu-samsung  I changed the sentence as you recommended and removed all the space between back-ticks and language identifier. 
I had fixed the conflict in TOC_all.